experiments/compositions/composition/Makefile

```diff
@@ -152,6 +157,30 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 	- $(CONTAINER_TOOL) buildx rm project-v3-builder
 	rm Dockerfile.cross
 
+##@ Testing
+
+.PHONY: release-kind-manifests
+release-kind-manifests: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	cd ../alice && $(KUSTOMIZE) build config/crd -o ../composition/release/kind/alice_crds.yaml
```

Comment:
Chatting with Justin and was wondering if we wanted to change alice to facade.

Reply:
can do in a subsequest change. It would be a set of small changes across a lot of files. Would prefer to do it after the e2e tests have been merged.